scheduleapp/templates/scheduleapp/add.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <h3>Add new activity:</h3>
+
+    <form method="post">{% csrf_token %}
+        {% if user.is_authenticated %}
```

Comment:
The whole form should be behind this condition. 
However, the view requires login so I don't think this if is necessary no?